Update article - Microsoft 365 group is supported (#103628)
Azure/azure-content: articles/active-directory/manage-apps/assign-user-or-group-access-portal.md

When you assign a group to an application, only users in the group will have access. The assignment doesn't cascade to nested groups.

> Group-based assignment requires Azure Active Directory Premium P1 or P2 edition. Group-based assignment is supported for Security groups and Microsoft 365 groups which SecurityEnabled is set to True only. Nested group memberships aren't currently supported. For more licensing requirements for the features discussed in this article, see the [Azure Active Directory pricing page](https://azure.microsoft.com/pricing/details/active-directory). 

Reviewer: ```suggestion
Group-based assignment requires Azure Active Directory Premium P1 or P2 edition. Group-based assignment is supported for Security groups and Microsoft 365 groups whose `SecurityEnabled` setting is set to `True` only. Nested group memberships aren't currently supported. For more licensing requirements for the features discussed in this article, see the [Azure Active Directory pricing page](https://azure.microsoft.com/pricing/details/active-directory). 
```
